sovtokenfees/sovtokenfees/static_fee_req_handler.py

```diff
@@ -147,6 +147,8 @@ def _fees_specific_validation(self, request: Request):
                                                        current_fees))
 
     def _validate_metadata(self, current_fees, constraint: AuthConstraint, wrong_aliases):
+        if constraint.constraint_id == ConstraintsEnum.FORBIDDEN_CONSTRAINT_ID:
```

Comment:
Please extend the tests in `sovtokenfees/test/authorize/metadata` (the same way as in indy-node for the Forbidden constraint)